### DISCUSSION TITLE
Extended statistics

### DISCUSSION THREAD

**Discussion by AlexanderRyb** [TriliumNext/Trilium#3733] [Ideas] [answered]
Have a way to find out things like: how many words/symbols were written in this folder total/ in this period of time. Bonus points for ability to make charts or other ways to represent data visually. Looks like a fun, moderately useful and straightforward to implement idea.

**Comment by zadam** [comment, marked as answer]
I plan to provide more statistic / graphs in the future. But things like "how many words/symbols were written in this folder total/ in this period of time" seems pretty custom and the best bet for that would be [scripting](https://github.com/zadam/trilium/wiki/Weight-tracker);
